index.html

```diff
@@ -197,12 +197,12 @@ <h2>Photo Gallery</h2>
                             </select>
 
                             <label>Do you like radio buttons?</label>
-                            <label><input type='radio' name='radio' value='Yes'>Yes</label>
-                            <label><input type='radio' name='radio' value='No'>No</label>
+                            <label><input type='radio' name='radio' value='yes'>Yes</label>
+                            <label><input type='radio' name='radio' value='no'>No</label>
```

Comment:
Even though HTML is generally case-insensitive, form values and names ARE case sensitive